helm/cluster-azure/templates/_externaldns_private_app.yaml

```diff
@@ -0,0 +1,69 @@
+{{/* Azure-specific external-dns app for private clusters */}}
```

Comment:
External DNS Private is special for Azure Private clusters we install it separate.